Civi/Test/Api3TestTrait.php

```diff
@@ -214,8 +218,8 @@ public function callAPISuccessGetSingle($entity, $params, $checkAgainst = NULL)
     ];
     $result = $this->civicrm_api($entity, 'getsingle', $params);
     if (!is_array($result) || !empty($result['is_error']) || isset($result['values'])) {
-      $unfilteredResult = $this->civicrm_api($entity, 'get', $params);
-      throw new \Exception(
+      $unfilteredResult = $this->civicrm_api($entity, 'get', ['version' => $this->_apiversion]);
```

Comment:
@eileenmcnaughton this seems to be problematic in that there maybe additional params passed in that we are no not passing through to the API call

Reply:
@seamuslee001 right - the point of this call was always to dump out what is in the DB - unfiltered - for that entity - but it wasn't working due to the fact it was being filtered